### DISCUSSION TITLE
Can't login to FreshRSS

### DISCUSSION THREAD

**Discussion by Stegemueller** [FreshRSS/FreshRSS#8323] [Q&A] [answered]
Hi there,

I really hope, you can help me. I've tried to install many times, and the installation is no problem.

After having completed the last step, I come to the login form, and enter the default username and the code. I keep getting this 403-oage saying 

Error 403 - Forbidden
You don’t have permission to access this page
← Go back to your RSS feeds

Can anything be done with this?

Best regards,
Hanne

**Comment by Alkarex** [comment]
Hello,
Could you tell a bit more about your platform (operative system, Web server, PHP version, FreshRSS version...)?

Try to reset the password manually with the following procedure:

1. Edit your file `./FreshRSS/data/config.php` and set `'auth_type' => 'none',`
2. In the Web interface of FreshRSS, go to Configuration / Profile, and change your password
3. In the Web interface of FreshRSS, go to Administration / Authentication, and change the Authentication method back to *Web Form*

In addition, check the logs:
* of FreshRSS, from the Web interface of FreshRSS
* of your Web browser console
* of your Web server

Possible errors include wrong file permissions on the server and JavaScript errors.

If login via Web Form cannot work for you, it is also possible to login via HTTP Auth.

**Comment by Alkarex** [comment]
@Stegemueller Did that fix the problem?

**Comment by Alkarex** [comment]
This issue could be related to https://github.com/FreshRSS/FreshRSS/issues/1807 (fixed), in particular if you were using double authentication (HTTP authentication and Web form).
Closing, but feel free to ask again :-)

**Comment by neuhaus** [comment]
You can 

1. use a website like https://keydecryptor.com/hash-tools/bcrypt or `htpasswd -nbBC 10 dummyuser my_tmp_password` to create a hash for a temporary password.
2. copy and paste the value that starts with `$2` to the `passwordHash` in the `config.php` of the user in the `users` subdirectory of the freshrss_data docker volume. For example `$2y$10$5kCWT8HlHToFSEw.aK5wZ.pTgShTYme7RmzM/ZVlpgsWJMWuTrFwq` for the password `my_tmp_password`.
3. log back in and set a new password using the web gui.

**Comment by Inverle** [comment, marked as answer]
Alternative:
```sh
docker exec -u www-data freshrss ./cli/update-user.php --user=[user] --password=[enter new password here]
```
or just `./cli/update-user.php --user=[user] --password=[enter new password here]` if not using Docker
